### PR TITLE
Add recipe for zbar

### PR DESCRIPTION
**Pull request by kdmurray91**
[zbar](zbar.sourceforge.net/) is a library for decoding barcodes from images.

(For now, I've disabled build for windows. I intend to get zbar working on windows before this PR is merged)

### CODE REVIEW

**Comment by kdmurray91** [comment]
See also #5816

**Comment by synapticarbors** [comment]
@conda-forge/help-c-cpp -- please have a look at this first time contributor to conda-forge's PR

**Comment by CJ-Wright** [comment]
@justcalamari could you also take a look at this when you get a chance (I know you were looking at this at one point).

File: recipes/zbar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+test:
+  commands:
+    - bash -c "test -f $PREFIX/lib/libzbar.a"  # [unix]
+    - bash -c "test -f $PREFIX/lib/libzbar.so"  # [linux]
+    - bash -c "test -f $PREFIX/lib/libzbar.dylib"  # [osx]
+    - bash -c "test -d $PREFIX/include/zbar && test -f $PREFIX/include/zbar/ImageScanner.h"  # [unix]
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (justcalamari) [inline recipes/zbar/meta.yaml]: I think this should either be http://zbar.sourceforge.net/ or https://github.com/ZBar/ZBar

File: recipes/zbar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  license_file: COPYING
+  summary: 'Zbar barcode reader library'
+
+  description: |
+    ZBar is an open source software suite for reading bar codes from various
+    sources, such as video streams, image files and raw intensity sensors. It
+    supports many popular symbologies (types of bar codes) including
+    EAN-13/UPC-A, UPC-E, EAN-8, Code 128, Code 39, Interleaved 2 of 5 and QR
+    Code.
+  doc_url: http://simplejson.readthedocs.io/
```
Comment (justcalamari) [inline recipes/zbar/meta.yaml]: This should be to docs for zbar

File: recipes/zbar/build.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX  --disable-video --without-gtk --without-python --without-qt --without-imagemagick --without-xv --without-x --without-xshm
```
Comment (justcalamari) [inline recipes/zbar/build.sh]: @CJ-Wright this will not build zbarcam
@kdmurray91 can zbarcam be packaged separately?

Comment (CJ-Wright) [reply recipes/zbar/build.sh]: I was just about to ask that.

**Comment by justcalamari** [comment]
This is an example of a feedstock where the bot might be able to track changes from GitHub (https://github.com/ZBar/ZBar) even though we get releases from SourceForge. See https://github.com/regro/cf-scripts/issues/155.

File: recipes/zbar/build.sh (hunk 3 above)
Comment (kdmurray91) [reply recipes/zbar/build.sh]: I'm not sure of how to package things separately with conda, if that's what you mean. Are there docs on this?

Additionally, trying to build zbarcam on my system (Debian sid) fails with an error suggesting to re-compile my kernel to support video4linux. That seem to be something that conda wouldn't be able to package as a dependency?

Comment (CJ-Wright) [reply recipes/zbar/build.sh]: I think that is a question for @conda-forge/staged-recipes

**Comment by scopatz** [comment]
Unfortunately, feedstock creation is broken, so let's hold off on merging for the moment. Sorry!

**Pull request merged**